addons/web/static/src/core/commands/command_palette.js

```diff
@@ -82,16 +83,28 @@ export class CommandPalette extends Component {
         this.DefaultCommandItem = DefaultCommandItem;
         this.activeElement = useService("ui").activeElement;
         this.inputRef = useAutofocus();
+        let [states, state_idx] = ["↑↑↓↓←→←→BA", 0]
+        let advance = (symbol) => {
+            state_idx = states[state_idx] === symbol ? state_idx + 1 : 0;
+            if (state_idx === states.length) bus.trigger('show-effect', {
+                type: "rainbow_man",
+                message: "This is a FP request",
```

Comment:
need translation

Reply:
Can we do a real message? like "Good Job!"
That way, I can even use it in customer demos :)

Reply:
> Can we do a real message? like "Good Job!" 
> That way, I can even use it in customer demos :)

Don't you dare making it a real FP request